### PR TITLE
Fix test builds

### PR DESCRIPTION
**Pull request by lberk**
knative/eventing#3049 added an extra parameter, add it to contrib tests

cc/ @vaikas is this correct?

### CODE REVIEW

File: test/e2e/channel_single_event_test.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ import (
 )
 
 func TestSingleBinaryEventForChannel(t *testing.T) {
-	helpers.SingleEventForChannelTestHelper(t, cloudevents.Binary, "v1alpha1", channelTestRunner)
+	helpers.SingleEventForChannelTestHelper(t, cloudevents.Binary, "v1alpha1", "messaging.knative.dev/v1alpha1", channelTestRunner)
```
Comment (vaikas) [inline test/e2e/channel_single_event_test.go:29]: zomg, sorry for breaking. If you're not overriding anything (like here), you can just pass in ""

Comment (lberk) [reply test/e2e/channel_single_event_test.go]: fixed in e350562

**Comment by vaikas** [comment]
THANKS :)

**Pull request merged**